ParkingSlots.Tests/Models/CreateViewModelTest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using ParkingZoneApp.Enums;
+using ParkingZoneApp.ViewModels.ParkingSlotViewModels;
+using System.ComponentModel.DataAnnotations;
+
+namespace ParkingSlots.Tests.Models;
+
+public class CreateViewModelTest
+{
+    
```

Comment:
where are create vm tests

Reply:
i have not moved to next US which is create action. And adding unit test to create vm will be implemented in the next US.